module/unicode/u8_textprep.c

```diff
@@ -358,8 +358,13 @@ u8_validate(const char *u8str, size_t n, char **list, int flag, int *errnum)
 	ret_val = 0;
 
 	no_need_to_validate_entire = ! (flag & U8_VALIDATE_ENTIRE);
+#if 0
 	check_additional = flag & U8_VALIDATE_CHECK_ADDITIONAL;
 	validate_ucs2_range_only = flag & U8_VALIDATE_UCS2_RANGE;
+#else
+	check_additional = 0;
+	validate_ucs2_range_only = 0;
+#endif
```

Comment:
Do we really need to be that petty? Code saving is insignificant, predictble branching may cost something, but I'd guess not much.

Reply:
commit dropped